### PR TITLE
[Beta] Delete unused variables

### PR DESCRIPTION
**Pull request by shamwela**
I did this using `yarn lint`.

### CODE REVIEW

File: beta/src/components/Button.tsx [hunk 1]
```diff
@@ -23,9 +23,9 @@ export function Button({
   return (
     <button
       style={style}
-      onMouseDown={(evt) => {
-        evt.preventDefault();
-        evt.stopPropagation();
+      onMouseDown={(event) => {
```
[the remark this answers is not in the document]
Comment (shamwela) [reply beta/src/components/Button.tsx:26]: I think it's more descriptive.

[the remark this answers is not in the document]
Comment (gaearon) [reply beta/src/components/Button.tsx:26]: Generally the convention is to use `e` which I'd prefer here. Avoids the mistake of accidentally forgetting to include the parameter name and then reading from the `event` global.

**Review by harish-sethuraman — APPROVED** [review]
LGTM

File: beta/src/components/Button.tsx [hunk 2]
```diff
@@ -23,9 +23,9 @@ export function Button({
   return (
     <button
       style={style}
-      onMouseDown={(evt) => {
-        evt.preventDefault();
-        evt.stopPropagation();
+      onMouseDown={(e) => {
```
Comment (harish-sethuraman) [inline beta/src/components/Button.tsx:26]: We could have left it as such may be?

Comment (shamwela) [reply beta/src/components/Button.tsx]: Ok.

**Review by harish-sethuraman — APPROVED** [review]

**Pull request merged**